/**:
  ros__parameters:
    lane_number: 3
    primitive_number: 4
    
    desired_lane: [2, 2, 2, 2]

    available_lanelet_id: [
                          6039, 8012, 6025,
                          6088, 8025, 6102,
                          5430, 8027, 7,
                          6204, 8029, 6218,
                          ]
    
    position:
      x: [
        45186.76745,  45184.14915,  45181.53085,
        45986.35145,  45983.70375,  45981.0894,
        45914.35565,  45917.83325,  45921.3732, 
        45405.89765,  45408.6405,   45411.2437,
        ]
        
      y: [
        45912.31385,  45909.9961,   45907.6735,
        45028.85555,  45026.4489,   45024.0585,
        44980.8065,   44980.86095,  44980.9021, 
        45638.76175,  45641.2157,   45643.59205,
        ]
      z: 0.0

    orientaion:
      x: 0.0
      y: 0.0
      z: [-0.7, 1.0, 1.0, 0.45]
      w: [0.7, 0.0, 0.0, 0.9]

    primitive_0: [
                6039,	8012,	6025,
                5832,	8018,	1682,
                6117,	8023,	6138,
                5149,	8024,	2322,
                6088,	8025,	6102,
                ]

    primitive_1: [
                6088,	8025,	6102,
                5326,	8026,	3482,
                5430,	8027,	7,
                ] 

    primitive_2: [
                5430,	8027,	7,
                6165,	8028,	6179,
                6204,	8029,	6218,
                ]

    primitive_3: [
                6204,	8029,	6218,
                5693,	8167,	1142,
                6039,	8012,	6025,
                ]

    # Slow On, Off를 위한 config
    curve_primitive: [
                  6117,	8023,	6138,
                  5149,	8024,	2322,
                  6165,	8028,	6179,
                  ]
    
    curve_primitive_1: [
                  6117,	8023,	6138,
                  5149,	8024,	2322,
                  ]

    curve_primitive_2: [
                  6165,	8028,	6179,
                  ]
    
    curve_1:
      position:
        x: [45133.4946,   45130.95845,  45128.4109]
        y: [45971.03685,  45968.6253,   45966.2242]
      orientaion:
        z: -0.7
        w: 0.7

    curve_2:
      position:
        x: [45915.51795,  45918.97905,  45922.4991]
        y: [44883.6361,   44883.68425,  44883.73325]
      orientaion:
        z: 0.7
        w: 0.7

    curve:
      position:
        z: 0.0
      orientation:
        x: 0.0
        y: 0.0
  
    # Go를 위한 config
    go_primitive: [2322, 6102]
    
    go:
      position:
        x: 45181.53085
        y: 45907.6735
        z: 0.0
      orientation:
        x: 0.0
        y: 0.0
        z: 0.0 # 수정
        w: 1.0 # 수정

    # Pit Stop을 위한 config
    pitstop_primitive: [6039,	8012,	6025,
                        8002, 8019, 8000,
                        8005, 8020, 8003,
                        ]
    
    pitstop:
      position:
        x: [45312.8491,   45315.46265,  45318.07725]
        y: [45741.65255,  45744.05395,  45746.4648]
        z: 0.0
      orientation:
        x: 0.0
        y: 0.0
        z: 0.37 # 수정
        w: 0.92 # 수정